tests/testthat/test-render.R

```diff
@@ -276,3 +276,47 @@ test_that("render_website() works - clean render for each language", {
   expect_true(fs::file_exists(file.path(parent_dir, project_dir, "_site", "fr", "subdir", "about.html")))
   expect_false(fs::dir_exists(file.path(parent_dir, project_dir, "_site", "es", "subdir")))
 })
+
```

Comment:
should there also be a test of putting links in the navbar for a book and in the sidebar for a website? for instance checking what the parent of the div with the right id is.

Reply:
maybe the helper functions for creating websites/books should gain a `placement` argument, then inside placement would be

```r
placement <- placement %||% switch(....
```

Reply:
> should there also be a test of putting links in the navbar for a book and in the sidebar for a website? for instance checking what the parent of the div with the right id is.

I can look into that.

> maybe the helper functions for creating websites/books should gain a `placement` argument, then inside placement would be
> 
> ```r
> placement <- placement %||% switch(....
> ```

That's a great idea. I'll work on it.

Reply:
I gave it a shot. Tell me if the tests are what you had in mind.

I added some checks to make sure the helper functions generate a `_quarto.yml` file that actually works "out of the box" depending on the placement choice.